docs/api/cli-options.md

```diff
@@ -37,6 +37,15 @@ Usage: start-storybook [options]
 Usage: build-storybook [options]
 ```
 
+<details>
+<summary><h4>Troubleshooting 6.0 Routing Issues</h4></summary>
+
+There is an [open issue (#11958)](https://github.com/storybookjs/storybook/issues/11958) in Storybook 6.0, which causes some built sites to have trouble with routing.
+
+If, after building your site, you cannot change your route using the sidebar, try building the site with the `--no-dll` as outlined [here](https://github.com/storybookjs/storybook/issues/11958#issuecomment-677726048).
+
+</details>
```

Comment:
@stabback we have a bit of room for improvements here and we probably could go with:
```md
<details>
    <summary><h4>Troubleshooting routing issues with Storybook 6.0</h4></summary>
     If you are building your Storybook and you encounter an issue where you cannot change the route in the sidebar, try building Storybook with the `--no-dll` flag and see if it solves the problem. If so, adjust your `build-storybook` script accordingly to include this flag. Be advised that the build process will run slower than usual.
</details>
```

Let me know what you think.

Reply:
Hey @jonniebigodes 

If you feel strongly about this I'm happy to update.  I'd like to leave the links in to provide context to anyone running into this issue.  If you'd rather not have the links to the issue I'm fine removing.

Reply:
@stabback like i've mentioned above there's always room for improvement. We can include the issue and phrase it like so:
```md
<details>
    <summary><h4>Troubleshooting routing issues with Storybook 6.0</h4></summary>
     If you are building your Storybook and you encounter an issue where you cannot change the route in the sidebar, try building Storybook with the `--no-dll` flag and see if it solves the problem. If so, adjust your `build-storybook` script accordingly to include this flag. We would like to point out that your build process will run slower than usual when using this flag. 

If you want, you can take a look at the following <a href="https://github.com/storybookjs/storybook/issues/11958"> issue </a> to get an in depth description of what is currently happening with your built Storybook.
</details>
```

Let me know if you're ok with this small adjustment here(sorry about the bad formatting) and adjust accordingly in the other document and we'll move from here.

Reply:
No problem at all, happy to keep things consistent with the rest of the documentation.  Updated both.

Reply:
@stabback thank you for applying the changes. This is good on my end. It can be merged.

Stay safe